api4/post.go

```diff
@@ -212,25 +212,20 @@ func getPostsForChannelAroundLastUnread(c *Context, w http.ResponseWriter, r *ht
 		return
 	}
 
-	etag := ""
 	if len(postList.Order) == 0 {
-		etag = c.App.GetPostsEtag(channelId)
```

Comment:
Honestly, I'm not sure how it happened to miss post.  Can't think of possible case. What if I change only the routeName from "Get Posts" to "Get Around Last Unread Posts" since params for c.App.GetPostsPage are really different between `getPostsForChannelAroundLastUnread` and `getPostsForChannel`? My hope is to change something now to see if it will make any difference.
```
if c.HandleEtag(etag, "Get Around Last Unread Posts", w, r)
```